docs/dashboards/detection-response-dashboard.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[detection-response-dashboard]]
+= Detection & Response dashboard
+
+The Detection & Response dashboard provides focused visibility into the day-to-day operational activities of your security environment. Security operations managers and analysts can quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with frequent and severe alerts.
+
+[role="screenshot"]
+image::detections/images/detection-response-dashboard.png[Overview of Detection & Response dashboard]
+
+Interact with various dashboard elements:
+
+* Use the date and time picker in the upper-right to specify a timeframe for displaying information on the dashboard. 
```

Comment:
```suggestion
* Use the date and time picker in the upper-right to specify a time frame for displaying information on the dashboard. 
```

Reply:
I've noticed we use `time range` more frequently when talking about the date and time range selected on the time picker but `time frame` works as well.

Reply:
I like "time range"!

Reply:
```suggestion
* Use the date and time picker in the upper-right to specify a time range for displaying information on the dashboard. 
```

Reply:
I like "time range"!
```suggestion
* Use the date and time picker in the upper-right to specify a time range for displaying information on the dashboard. 
```